Frontend/CO.CDP.OrganisationApp.Tests/Pages/Organisation/OrganisationEmailModelTest.cs

```diff
@@ -60,12 +84,33 @@ public void WhenEmailAddressIsInvalid_ShouldRaiseEmailAddressValidationError()
         var model = GivenOrganisationEmailModel();
         model.EmailAddress = "dummy";
 
-        var results = ModelValidationHelper.Validate(model);
+        var stringLocalizerMock = new Mock<IStringLocalizer>();
+        stringLocalizerMock
+            .Setup(localizer => localizer[nameof(StaticTextResource.Global_Email_Invalid_ErrorMessage)])
+            .Returns(new LocalizedString(nameof(StaticTextResource.Global_Email_Invalid_ErrorMessage), StaticTextResource.Global_Email_Invalid_ErrorMessage));
+
+        var stringLocalizerFactoryMock = new Mock<IStringLocalizerFactory>();
+        stringLocalizerFactoryMock
+            .Setup(factory => factory.Create(It.IsAny<Type>()))
+            .Returns(stringLocalizerMock.Object);
+
+        var serviceProviderMock = new Mock<IServiceProvider>();
+        serviceProviderMock
+            .Setup(provider => provider.GetService(typeof(IServiceProvider)))
+            .Returns(serviceProviderMock.Object);
+
+        serviceProviderMock
+            .Setup(provider => provider.GetService(typeof(IStringLocalizerFactory)))
+            .Returns(stringLocalizerFactoryMock.Object);
+
+        var validationContext = new ValidationContext(model, serviceProviderMock.Object, null);
```

Comment:
All this validationContext and service provider setup looks very similar in all these test methods. Could we extract it and parametrise? It would make the test case shorter and if named properly, let us understand what's going on here.

Reply:
done.